jazz_environment-event-handler/test/test.js

```diff
@@ -181,11 +277,169 @@ describe('jazz environment handler tests: ', () => {
+		});
+
+		index.processEventUpdateEnvironment(environmentPayload, configData, authToken)
+		.catch(res => {
+			sinon.assert.calledOnce(requestPromiseStub);
+			requestPromiseStub.restore();
+			expect(res.details).to.eql('Error');
+		})
+	});
+
+	it('Verify processEventCreateBranch rejects with statusCode of 400', () => {
+		let environmentPayload = testPayloads.environmentPayload;
+		environmentPayload.logicalId = "6knr9d33tt-dev";
+
+		let requestPromiseStub = sinon.stub(request, "Request", (obj) => {
+			return obj.callback(null, testPayloads.createBranchError, JSON.stringify(testPayloads.createBranchError.body));
+		});
+
+		index.processEventCreateBranch(environmentPayload, configData, authToken)
+		.catch(res => {
+			sinon.assert.calledOnce(requestPromiseStub);
+			requestPromiseStub.restore();
+			expect(res.details).to.eql('error');
+		})
+	});
+
+	it('Verify processEventInitialCommit rejects with statusCode of 400', () => {
+		let environmentPayload = testPayloads.environmentPayload;
+
+		let requestPromiseStub = sinon.stub(request, "Request", (obj) => {
+			return obj.callback(null, testPayloads.processEventInitialCommitError, JSON.stringify(testPayloads.processEventInitialCommitError.body));
+		});
+
+		index.processEventInitialCommit(environmentPayload, configData, authToken)
+		.catch(res => {
+			sinon.assert.calledTwice(requestPromiseStub);
+			requestPromiseStub.restore();
+			expect(res.details).to.eql('error');
+		});
+	});
+
+	it('Verify processEventInitialCommit rejects when physical id is different', () => {
+		let environmentPayload = testPayloads.environmentPayload;
+		environmentPayload.physical_id = "physicalId";
+		
+		index.processEventInitialCommit(environmentPayload, configData, authToken)
+		.catch(res => {
+			expect(res).to.eql('INITIAL_COMMIT event should be triggered by a master commit. physical_id is physicalId');
+		});
+	});
+
+	it('Verify processEventInitialCommit resolves with statusCode of 200', () => {
+		let environmentPayload = testPayloads.environmentPayload;
+		environmentPayload.physical_id = 'bugfix/test_02';
+		let requestPromiseStub = sinon.stub(request, "Request", (obj) => {
+			return obj.callback(null, testPayloads.processEventInitialCommitSuccess, testPayloads.processEventInitialCommitSuccess.body);
+		});
+
+		index.processEventInitialCommit(environmentPayload, configData, authToken)
+		.then(res => {
+			sinon.assert.calledTwice(requestPromiseStub);
+			requestPromiseStub.restore();
+			expect(res.message).to.eql('Stage and Prod environments are created successfully');
+		});
+	});
+
+	it('Verify processEventDeleteBranch rejects when unable to delete environment', () => {
+		let environmentPayload = testPayloads.environmentPayload;
+		environmentPayload.physical_id = 'bugfix/test_02';
+
+		let requestPromiseStub = sinon.stub(request, "Request", (obj) => {
```

Comment:
What part of this unit test case is setting up the condition to _fail delete environment_?